package.json

```
@@ -19,7 +19,7 @@
     "test:coverage": "live-server ./coverage/lcov-report",
     "test:coverage-branch": "npm run test:unit -- --coverage && git diff origin/develop | diff-test-coverage -c ./coverage/lcov.info -t lcov --",
     "test:e2e:browserstack": "nightwatch -c tests/e2e/browserstack.conf.js --suiteRetries 3",
-    "test:e2e:local": "nightwatch --config ./tests/e2e/local-chrome.conf.js",
+    "test:e2e:local": "npx kill-port 9515 && nightwatch --config ./tests/e2e/local-chrome.conf.js",
```

```suggestion
    "test:e2e:local": "nightwatch --config ./tests/e2e/local-chrome.conf.js",
```
this doesn't work properly on Windows

---

sorry, didn't mean to leave this in. was an issue every time I ran e2e.